cosmic/tests/test_utils.py

```diff
@@ -33,14 +33,14 @@
                'binary_state' : [0]}
 false_dict = {'select_final_state' : False,
              'binary_state' : [0,1,2]}
-conv_dict_formation = {'convergence_filter' : 'formation'}
-conv_dict_1_SN = {'convergence_filter' : '1_SN'}
-conv_dict_2_SN = {'convergence_filter' : '2_SN'}
-conv_dict_disruption = {'convergence_filter' : 'disruption'}
-conv_dict_final_state = {'convergence_filter' : 'final_state'}
-conv_dict_XRB_form = {'convergence_filter' : 'XRB_form'}
-conv_dict_false = {'convergence_filter' : 'wrong'}
```

Comment:
I think Scotty changed these all to 'convergence_filter' in the recent PR, so I think you may have to change yours so there isn't a conflict. Is that right @scottcoughlin2014?